pages/collection/[chain]/[contract]/[id].tsx

```diff
@@ -478,9 +485,10 @@ const IndexPage: NextPage<Props> = ({ id, collectionId, ssr }) => {
                 collectionAttributes={attributesData?.data}
               />
               <PriceData token={token} />
-              {isMounted && (
+              {isMounted && !offersLoading && (
```

Comment:
Hm not sure we want to do this, block all of the actions because the offers haven't been loaded yet?